Index: src/main/java/com/github/zly2006/reden/debugger/stages/block/StageBlockNCUpdateSixWay.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/github/zly2006/reden/debugger/stages/block/StageBlockNCUpdateSixWay.kt b/src/main/java/com/github/zly2006/reden/debugger/stages/block/StageBlockNCUpdateSixWay.kt
--- a/src/main/java/com/github/zly2006/reden/debugger/stages/block/StageBlockNCUpdateSixWay.kt	
+++ b/src/main/java/com/github/zly2006/reden/debugger/stages/block/StageBlockNCUpdateSixWay.kt	(date 1702183043059)
@@ -1,5 +1,4 @@
 package com.github.zly2006.reden.debugger.stages.block
-import com.github.zly2006.reden.access.TickStageOwnerAccess
 import com.github.zly2006.reden.debugger.TickStage
 import com.github.zly2006.reden.utils.readBlock
 import com.github.zly2006.reden.utils.readDirection
